modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -53,7 +53,7 @@ package object dao extends MoreTupleOps with ToUserProgramRoleOps {
     Meta[List[Int]].nxmap(Location.unsafeMiddle(_), _.toList)
 
   implicit val DurationMeta: Meta[Duration] =
-    Meta[Long].xmap(Duration.ofSeconds, _.getSeconds)
+    Meta[Long].xmap(Duration.ofMillis, _.toMillis)
```

Comment:
Right, you can add [custom mappings](https://tpolecat.github.io/doobie-0.2.1/10-Custom-Mappings.html) for your types as explained in the Book of Doobie.